ui/static/templates/databases.html

```diff
@@ -35,6 +35,12 @@
                         {% endfor %}
                     </div>
                 {% endif %}
+                
+                <div class="mb-4">
+                    <button id="back-button" type="button" class="btn btn-red mr-auto" onclick="window.location.href='http://127.0.0.1:8000/';">
```

Comment:
statt onclick sollte auch nur href="/" genügen, dass wäre dann auch mit line 4 in navbar.html konsistent sein. 
```suggestion
                    <button id="back-button" type="button" class="btn btn-red mr-auto" href="/">
```

Reply:
Habe gerade mal nachgeschaut das `href="/"` scheint nur mit dem `<a>-Tag` zu gehen und wurde so auch z. B. in `details.html` verwendet. Passe das entsprechend an, danke!